dags/map_reproducibility/nemo_gpt3.py

```diff
@@ -46,6 +46,7 @@ def run_aotc_workload():
   helm_cmds = (
       "CONFIG_FILE=$REPO_ROOT/src/frameworks"
       "/nemo-configs/gpt3-175b-256gpus-fp8.yaml",
+      "export JOB_NAME=gpt3-xlml-$NOW-175b-nemo",
```

Comment:
can we put workload specific variables into another function, eg. set_workload_variable_cmds